thrust/thrust/detail/config/forceinline.h

```diff
@@ -32,7 +32,7 @@
 
 #if defined(__CUDACC__) || defined(_NVHPC_CUDA)
 
-#define __thrust_forceinline__ __forceinline__
+#define __thrust_forceinline__ _CCCL_FORCEINLINE
```

Comment:
important: `_CCCL_FORCEINLINE` is not defined in this header. I guess you need to include cccl config.

Reply:
I removed the whole header, replacing `__thrust_forceinline__ ` with _CCCL_FORCEINLINE